src/notifications/tests/test_models.py

```diff
@@ -63,3 +65,90 @@ def new_notification(sender, **kwargs):
 
         notes_unread = Notification.objects.all_unread(self.student)
         self.assertEqual(notes_unread.count(), 1)
+
+    def test_notification_str(self):
+        """ 
+            Check if notification string method leaves behind un-stripped html tags except <img>.
+        """ 
+        # CONSTANTS
```

Comment:
These constants should be in a `setUp(self)` method, see how other test classes use setUp.  The setup method is run before each individual test and ensures a predictable state for each test, and prevents data from changing between tests, and so that test order shouldn't matter.

Reply:
Hmm... I didn't realize you are running methods inside methods.  I think this might be more standard and easier to read if you just created your own test class for testing this method.  The boiler plate helper method can be it's own method, or just run it or put it in setUp.  The goal is for each specific thing you are testing to have it's own test method, as precise as possible.  I know in this project there are many waaay to big tests... those are old and bad and need to be broken up. So best practices as best we can from here on in!

Reply:
put it in its own test class